### ISSUE TITLE
[ToDo] Use forthcoming strings.ToValidUTF8

### ISSUE THREAD

**Issue by ymmt2005** [cybozu-go/log#19]
https://github.com/golang/go/issues/25805

This library uses `string([]rune(s))` technique to remove invalid Unicode characters
in multiple places.  They should be replaced with `strings.ToValidUTF8` which will
be added in Go 1.13.

```console
$ git grep '\[\]rune'
json.go:                        t = string([]rune(t))
json.go:                        s = string([]rune(s))
logfmt.go:                      t = string([]rune(t))
logfmt.go:                      s = string([]rune(s))
plain.go:                       t = string([]rune(t))
plain.go:                       s = string([]rune(s))
```

**Comment by ymmt2005** [comment]
`strings.ToValidUTF8` will be added to Go 1.13
https://github.com/golang/go/commit/3259bc441957bf74f069cf7df961367a3472afb2

**Comment by Hsn723** [comment]
Closing as this was implemented in #26 and released as v1.6.0
